.travis.yml

```diff
@@ -2,13 +2,18 @@ language: php
 
 matrix:
     include:
-        - php: 5.6
-        - php: 7.0
-          env: lint=1
         - php: 7.1
           env: DEPENDENCIES='dev' lint=1
         - php: 7.1
           env: DEPENDENCIES='low'
+        - php: 7.2
+          env: DEPENDENCIES='dev' lint=1
```

Comment:
there is no reason to have multiple jobs performing the CS linting. That linting does not depend on the PHP version.

Reply:
Updated